gcloud-java-storage/src/main/java/com/google/cloud/storage/testing/package-info.java

```diff
@@ -32,6 +34,21 @@
  * RemoteStorageHelper.forceDelete(storage, bucket, 5, TimeUnit.SECONDS);
  * } </pre>
  *
+ * LocalGcsHelper helps with testing on an in-memory filesystem (this is best for unit tests).
```

Comment:
Actually we should remove this as we hid `LocalStorageHelper`. Good that we spotted this!